build/docker/grafana/dashboard.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: 1
```

Comment:
Is this manifest needed even if we need to create/import dashboard ourselves?

Reply:
The dashboard configuration file is not needed if we choose to create or import dashboards manually. Given your question about its significance, I've reconsidered its usefulness. Upon reflection, I agree that the file may not offer substantial advantages. As a result, I've decided to remove it and have committed this change.